karma.conf.js

```diff
@@ -68,7 +68,9 @@ module.exports = function(karma) {
 			plugins: [
 				resolve(),
 				babel({envName: 'test', exclude: 'node_modules/**'}), // use babel since we have ES proposal features
-				commonjs()
+				commonjs({
+					include: "node_modules/**"
```

Comment:
Why is this needed? Doesn't it take a lot of time?

Reply:
I believe the only thing `commonjs` is needed for is pixelmatch. I was trying to make it clear that is wasn't used in our own code. I've updated it to use `exclude` see if that makes it any clearer

Reply:
ok, so it works without exclude/include the same?

Reply:
I think the `include` and `exclude` I had were the same. If you have neither it will try to process our src and test files